### ISSUE TITLE
Code Prettify shortcut interferes with Sublime keymap

### ISSUE THREAD

**Issue by elmerehbi** [ipython-contrib/jupyter_contrib_nbextensions#1105]
Ctrl+Shift+L is being used by Code Prettify which interferes with the Sublime keymap for multi-line editing.

When trying to multi-line edit using this shortcut an error pops up. The code is also pretified.
Here's the error:

`
[code_prettify]
 Error: SyntaxError
invalid syntax (<unknown>, line 1)
`

**Comment by jcb91** [comment]
The shortcut for code prettify is (should be!) editable - the [default is ctrl-shift-L](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/blob/862c6b311328ab80c09c2b4d4e56af56de6fc36e/src/jupyter_contrib_nbextensions/nbextensions/code_prettify/code_prettify.yaml#L42), but you should be able to change this from the nbextensions configurator. Have you tried altering it to something else?

**Comment by elmerehbi** [comment]
@jcb91 Yes I actually removed the shortcut but I though I'd report it anyways. Also I found it quite cumbersome to reset the shortcut. More specifically the hotkey captured is: `Ctrl, Ctrl-Shift-L`.

**Comment by jcb91** [comment]
@elmerehbi I think perhaps I do not understand exactly the problem you are having.

> Also I found it quite cumbersome to reset the shortcut

I'm not sure what you mean, could you explain a little about what process would you have preferred, in contrast to what you got?

> More specifically the hotkey captured is: `Ctrl, Ctrl-Shift-L`

I'm not sure what you mean by this? Do you mean that the configurator is not capturing correctly the hotkey you wanted to use for code-prettify?

I guess perhaps we could try to alter the default code-prettify, but it gets difficult to find enough combos for everything :sweat_smile:
